fia_auth/tokens.py

```diff
@@ -69,15 +69,15 @@ class AccessToken(Token):
     def __init__(self, jwt_token: str | None = None, payload: dict[str, Any] | None = None) -> None:
         if payload and not jwt_token:
             self._payload = payload
-            self._payload["exp"] = datetime.now(UTC) + timedelta(minutes=5)
+            self._payload["exp"] = datetime.now(UTC) + timedelta(minutes=10)
```

Comment:
Is it possible to make this an environment variable? defaulting to 10 minutes, we don't even need to change it, but it will potentially be useful in the long run.